src/main/java/gtPlusPlus/xmod/gregtech/recipes/GregtechRecipeAdder.java

```diff
@@ -599,6 +599,59 @@ public boolean addBlastSmelterRecipe(
         return aSize > aSize2;
     }
 
+    @Override
+    public boolean addQuantumTransformerRecipe(
+            ItemStack[] aInput,
+            FluidStack[] aFluidInput,
+            FluidStack[] aFluidOutput,
+            ItemStack[] aOutputStack,
+            int[] aChances,
+            int aDuration,
+            int aEUt,
+            int aSpecialValue) {
+        if ((aInput == null)) {
+            Logger.WARNING("Fail - Input was null.");
+            return false;
+        }
+        for (int i = 0; i < aFluidOutput.length; i++) {
+            if ((aDuration = GregTech_API.sRecipeFile.get(
+                            "quantumsmelter", aFluidOutput[i].getFluid().getName(), aDuration))
+                    <= 0) {
+                Logger.WARNING("Recipe did not register.");
+                return false;
+            }
+        }
+
+        for (int das = 0; das < aInput.length; das++) {
```

Comment:
Whats `das`? I'd suggest either using the standard `i` or a more explanatory name.